pkg/pwgen/memorable.go

```diff
@@ -4,16 +4,34 @@ import "strings"
 
 // GenerateMemorablePassword will generate a memorable password
 // with a minimum length
-func GenerateMemorablePassword(minLength int, symbols bool) string {
+func GenerateMemorablePassword(minLength int, symbols bool, capitals bool) string {
```

Comment:
That's a public API change, but it seems only Gopass and Gopass JSON-API (which don't use that function) is using this package currently, so that should be fine.
We might want to increment a minor version instead of a patch version because of that tho.

Reply:
We should probably make it a minor bump, yes. But otherwise I wouldn't be concerned about this change. This package is only used by gopass itself and gopass-jsonapi: https://pkg.go.dev/github.com/gopasspw/gopass@v1.12.8/pkg/pwgen?tab=importedby

That being said I also wouldn't set the bar too high here. This package doesn't have a very good interface anyway and is likely due for a major redesign at some point. The interfaces aren't very consistent. Cryptic uses an object while the others can only be called directly. Either all methods should be static or all object methods where the configuration goes into the object. Maybe with static helpers that use the most common settings, like the http package does.